test/templates/input-group/index.njk

```diff
@@ -5,275 +5,133 @@
 {% from "./../format.njk" import format %}
 
 {% block content %}
-<style media="screen">
-    .fd-input-group {
-        margin-bottom: 20px;
-    }
-</style>
 
-<h1>w/ text addon</h1>
-{% set example %}
+{% set types = ["text", "textarea"] %}
```

Comment:
Changed most of this file to use for loops to generate multiple examples rather than all being hard-coded.